gamesh:
  enable_identified_points: false
  enable_inverse_conic: true
  enable_points_position_update: true
  enable_ray_mistrust: false
  enable_unused_vertex_removing: false
  enable_mesh_saving: true
  enable_mesh_publishing: true
  generate_colored_mesh: true
  
  free_vote_threshold: 20.0
  non_conic_free_vote_threshold: 1
  ray_removal_threshold: 2.0
  unused_vertex_removal_threshold: 100000000.0
  primary_points_visibility_threshold: 1
  
  max_points_per_camera: 100
  max_distance_camera_points: 40.0
  steiner_grid_step_length: 1.0
  w_1: 10.0
  w_2: 2.0
  w_3: 1.0
  w_m: 1.0
  
  triangulation_update_every: 1
  initial_triangulation_update_skip: 2
  save_mesh_every: 1
  
  time_stats_output: true
  debug_output: false
  publish_received_pointcloud: true
  publish_used_pointcloud: true
  
  input_topic: "gamesh_rays"
  output_topic: "gamesh_mesh"
  
  received_pointcloud_topic: "gamesh_bridge_points"
  used_pointcloud_topic: "gamesh_points"
  
  output_folder: "/home/enrico/gamesh_output/"
  time_stats_folder: "/home/enrico/gamesh_stats/"
  count_stats_folder: "/home/enrico/gamesh_stats/"
